setup.py

```diff
@@ -178,9 +178,7 @@ def check_for_openmp():
         "url": 'https://khmer.readthedocs.org/',
         "packages": ['khmer', 'khmer.tests', 'oxli'],
         "package_dir": {'khmer.tests': 'tests'},
-        "install_requires": ['screed >= 0.9'],
-        # testing screed download link
-
+        "install_requires": ['screed >= 0.9', 'bz2file'],
```

Comment:
On Fri, Jul 31, 2015 at 07:25:26AM -0700, Michael R. Crusoe wrote:
> > @@ -178,9 +178,7 @@ def check_for_openmp():
> >          "url": 'https://khmer.readthedocs.org/',
> >          "packages": ['khmer', 'khmer.tests', 'oxli'],
> >          "package_dir": {'khmer.tests': 'tests'},
> > -        "install_requires": ['screed >= 0.9'],
> > -        # testing screed download link
> > -
> > +        "install_requires": ['screed >= 0.9', 'bz2file'],
> 
> `bz2file` is already a requirement of screed

ok, same comment applies there :). @bocajnotnef punt to an issue if
you want.